docs/installation.rst

```diff
@@ -258,9 +259,10 @@ To check that you have properly installed DESC and its dependencies, try the fol
     python
     >>> from desc import set_device  # only needed if running on a GPU
     >>> set_device('gpu')  # only needed if running on a GPU
+    >>> from desc.backend import print_backend_info
+    >>> print_backend_info()
     >>> import desc.equilibrium
```

Comment:
i think in theory we can get rid of this line, previously it was only meant to trigger the backend import to print the info, but thats its own call now